plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveBucketingV1.java

```diff
@@ -38,11 +38,10 @@
 {
     private HiveBucketingV1() {}
 
-    static int getBucketHashCode(List<TypeInfo> types, Page page, int position)
+    static int getBucketHashCode(List<TypeInfo> types, Page page, int prefixChannels, int position)
```

Comment:
what's relation between `types.size()` and `prefixChannels`?

Reply:
added checks in https://github.com/trinodb/trino/pull/6970